docker/sandbox/sandbox-sus2.yaml

```diff
@@ -60,18 +60,9 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: mediawiki-sandbox-ingress
-  annotations:
-    traefik.frontend.priority: "1"
 spec:
   rules:
-    - host: "mediawiki.prod.sjc.k8s.wikia.net"
-      http:
-        paths:
-          - path: /
-            backend:
-              serviceName: mediawiki-sandbox
-              servicePort: 80
-    - host: "*.wikia.com"
+    - host: "sandbox-sus2.prod.sjc.k8s.wikia.net"
```

Comment:
How this will works? We need to override host on border to make it works. And then use original-host header